mix.exs

```diff
@@ -116,7 +123,8 @@ defmodule Franklin.MixProject do
       "event_store.reset": ["event_store.drop", "event_store.setup"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
       test: ["ecto.create --quiet", "ecto.migrate --quiet", "test"],
-      "assets.deploy": ["tailwind default --minify", "esbuild default --minify", "phx.digest"]
+      "assets.deploy": ["tailwind default --minify", "esbuild default --minify", "phx.digest"],
+      lint: ["credo --strict", "dialyzer --quiet"]
```

Comment:
```suggestion
      lint: ["credo --strict", "dialyzer"]
```

I think my preference would be to drop the `--quiet` flag, as when I run `mix lint` locally, it pauses for a good while without any real output and that might cause confusion as to what is going on.

I also saw in your earlier link they were using `mix dialyzer --format github`. I'm not sure how that flag would impact my CI use but noting it in case I want to come back to this in the future.